src/NHibernate.Test/NHSpecificTest/NH1812/Fixture.cs

```diff
@@ -1,20 +1,26 @@
 using System;
-using System.Collections.Generic;
 using NUnit.Framework;
 
 namespace NHibernate.Test.NHSpecificTest.NH1812
 {
-	public class AstBugBase : BugTestCase
+	public class AstBug : BugTestCase
```

Comment:
This test was originally having:

-  `AstBugBase` class without a `Fixture` attribute, not meant to be run (should have been further marked `abstract` for guaranteeing this).
- `AstBug`, just deriving from `AstBugBase` with a `Fixture` attribute, meant to demonstrate the bug.
- `ItWorksWithClassicParser`, deriving from `AstBugBase` with a `Fixture` attribute and alter the configuration for using the "classic parser". Meant to demonstrate this was a regression.

The later has been removed, rendering the inheritance scheme moot. But that was not cleaned. And NUnit changes caused the "base" class to be run too.

I have cleaned up that.